src/runtime_src/core/common/api/kernel_int.h

```diff
@@ -37,8 +37,8 @@ namespace xrt_core { namespace kernel_int {
 void
```

Comment:
The copyright date (line 2) needs to be updated.

Reply:
Fixed.